starlark/interp.go

```diff
@@ -590,6 +593,21 @@ loop:
 	return result, err
 }
 
+type loadError struct {
```

Comment:
Is this necessary? I think EvalError should suffice.

Reply:
wrapping this as an EvalError isn't quite right, because (as you pointed out at the start of this review) EvalError has a backtrace. That backtrace would be wrong here, and makes the existing test TestThreadLoad_ParallelCycle fail

Reply:
Right, but I meant its CallStack field would be nil.  Actually, probably best not to, since the starlark.Call checks that all returned errors are EvalError, and it would fail to re-wrap the load error in that case.

OK, let's keep a distinct type, but let's declare it thus:
```
type wrappedError struct {
    msg string
    cause err
}
```

Reply:
done!